cluster/v1/types.go

```diff
@@ -96,6 +96,9 @@ const (
 	// SpokeClusterConditionHubDenied means the request to join the cluster is denied by
 	// cluster-admin on hub
 	SpokeClusterConditionHubDenied string = "HubDeniedSpoke"
+	// SpokeClusterConditionAPIServerAvailable means the kube-apiserver of the spoke cluster is
+	// available
+	SpokeClusterConditionAPIServerAvailable string = "SpokeClusterAPIServerAvailable"
```

Comment:
Is this really testing whether the API is available, or whether the spoke agent is running?

Reply:
I plant to use `healthz` API to check the apiserver status, I think if the `healthz` API returns ok, I can consider the apiserver is available, and I check the apiserver status in agent, so I think if I can check the status, the spoke agent should be running